docs/section23/Rule23-2.md

```diff
@@ -17,26 +17,23 @@
 **Function Calls:**  
 
 1. get_baseline_system_types()
-2. is_baseline_system_type()
+2. baseline_system_type_compare()
 
 **Applicability Checks:**  
-
+- create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_5,HVAC_SYS.SYS_6,HVAC_SYS.SYS_7,HVAC_SYS.SYS_8,HVAC_SYS.SYS_11]`
 - Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
 
-  - Check if B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8, 11.1, 11.2, 7a, 8a, 11.1a, 11.2a, 5b, 6b, 7b, 8b, 11b, 7c, 11c, continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-5", "SYS-6", "SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-7A", "SYS-8A", "SYS-11.1A", "SYS-11.2A", "SYS-5B", "SYS-6B", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-7C", "SYS-11C"]): CHECK_RULE_LOGIC`
+  - Check if B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8, 11.1, 11.2, 7a, 8a, 11.1a, 11.2a, 5b, 6b, 7b, 8b, 11b, 7c, 11c, continue to rule logic: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types): CHECK RULE LOGIC`
 
   - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
 
 ## Rule Logic:  
 
-- For each building segment in B_RMR: `for building_segment_b in B_RMR...building_segments:`
-
-  - For each HVAC system in building segment: `for hvac_b in building_segment_b.heating_ventilating_air_conditioning_systems:`
-
-    - Get HVAC system serving terminal: `hvac_b = terminal_b.served_by_heating_ventilating_air_conditioning_system`
-  
-      - Check if HVAC system is type 5, 7, 7a, 5b, 7b, 7c, 6, 8, 8a, 6b, 8b: `if any(is_baseline_system_type(hvac_b, sys_type) == TRUE for sys_type in ["SYS-5", "SYS-7", "SYS-7A", "SYS-5B", "SYS-7B", "SYS-7C", "SYS-6", "SYS-8", "SYS-8A", "SYS-6B", "SYS-8B"]):`
-
+- For each hvac system type in the building: `for baseline_system_type in baseline_hvac_system_dict:`
```

Comment:
I am not sure what this line is doing.

Reply:
the function `get_baseline_system_types`, called on line 24, returns a dictionary in the format:

{"SYS-3": ["hvac_id_1", "hvac_id_10"], "SYS-7A": ["hvac_id_3", "hvac_id_17", "hvac_id_6], "SYS-9": ["hvac_id_2"]}

line 32 iterates through the system types (ie "SYS-3" in the example above).  Calling `baseline_hvac_system_dict[baseline_system_type]` returns a list of hvac_ids of that system type.